src/fuzz_introspector/code_coverage.py

```diff
@@ -221,6 +235,7 @@ def load_llvm_coverage(
                 found_name = target_name
 
     cp = CoverageProfile()
+    cp.set_type("file")
```

Comment:
This should be "function" or something of this nature, but not "file", right?

Reply:
Oops my bad, this was intended to be removed. Thanks for pointing out.

Btw, `function` type sounds good. I will use it.

Reply:
awesome!